internal/pkg/service/buffer/api/service/service.go

```diff
@@ -41,7 +41,7 @@ func New(d dependencies.ForServer) buffer.Service {
 	return &service{
 		deps:   d,
 		mapper: mapper.NewMapper(d.BufferAPIHost()),
-		stats: statistics.New(d.Process().Ctx(), func(ctx context.Context, stats []model.SliceStats) {
+		stats: statistics.New(d.Process().Ctx(), d, func(ctx context.Context, stats []model.SliceStats) {
```

Comment:
V tomto commite som pridal `Clock()` do base dependencies:
- pouziva to aj watch kod
- bude sa to dat testovat cez mockedDependencies